topbeat/beater/config.go

```diff
@@ -1,17 +1,48 @@
 package beater
 
+import (
+	"errors"
+	"time"
+)
+
+type ConfigSettings struct {
```

Comment:
yes, one can uses nested paramters. But due to topbeat trying to figure out the right section this won't work. `ConfigSettings` being obsolete, how about:

```
type Config struct {
  Period time.Duration    `config:"period"  validate:"min=1ms"`
  Procs []string          `config:"procs"`
  Stats StatsEnableConfig `config:"stats"`
}
```

The beater already finds and unpacks from correct child section.